### DISCUSSION TITLE
any plan for submenu?

### DISCUSSION THREAD

**Discussion by sunkangchina** [filamentphp/filament#336] [Ideas]
now left menu just  one level ,not support submenu,but in a project ,has many menus,that mean needs submeun.

**Comment by ManojKiranA** [comment]
Now its possible

https://github.com/filamentphp/filament/pull/4650

**Reply by danharrin** [reply]
Or just with navigation groups... this thread was made in April 2021

**Reply by ManojKiranA** [reply]
I have put that here because the PR is not yet documented so I thought it would be ok to leave a comment here.

**Reply by ManojKiranA** [reply]
And usually I will go through all the discussion once in a while so that I can help others.

**Reply by ManojKiranA** [reply]
I think few days or week back you have updated the table builder docs. It's very nice 😊
